sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsLifecycleSamples.cs

```diff
@@ -217,12 +219,18 @@ await foreach (IncomingRelationship incomingRelationship in incomingRelationship
                     }
 
                     // Now the digital twin should be safe to delete
-                    string digitalTwinId = twin.Key;
 
                     #region Snippet:DigitalTwinsSampleDeleteTwin
 
-                    Response deleteDigitalTwinResponse = await client.DeleteDigitalTwinAsync(digitalTwinId);
-                    Console.WriteLine($"Deleted digital twin with Id {digitalTwinId}. Response Status: {deleteDigitalTwinResponse.Status}");
+                    try
```

Comment:
I think the reason behind the big try catch block was to remove the try catch from the snippet region, let's stick to that if possible.

Reply:
sure

Reply:
correct me if I am wrong here @drwill-ms

Reply:
I agree, doesn't make sense to enclose a try-catch within another try-catch block, for our scenario.